models/cross-spork_client.go

```diff
@@ -0,0 +1,142 @@
+package models
```

Comment:
Minor: Can we rename the file to `cross_spork_client.go`. Right now there's mixed `-` & `_`.
Another note: The functionality of this `CrossSporkClient` doesn't make me feel like it belongs to the `models` package, but rather the `services` package. What do you think?

Reply:
Oh, I see. This didn't even cross my mind :innocent: